xen/drivers/passthrough/Makefile

```diff
@@ -7,3 +7,4 @@ obj-y += iommu.o
 obj-$(CONFIG_X86) += io.o
```

Comment:
workaround to comment on commit message:
"It was considered to keep two sets of API+callback" - it s not clear for the rest of the world who and when have considered that, so may the statement generic.